src/explore-education-statistics-admin/src/components/AdminDashboardPublicationsTab.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Dictionary } from '@common/types';
+import Accordion from '@common/components/Accordion';
+import AccordionSection from '@common/components/AccordionSection';
+import { Publication } from '@admin/services/publicationService';
+import AdminDashboardPublications from './AdminDashboardPublications';
+import Link from './Link';
+
+interface AdminDashboardPublicationsTabProps {
+  publicationsByThemeAndTopic: Dictionary<Publication[]>;
+  noResultsMessage: string;
+}
+
+const AdminDashboardPublicationsTab = ({
+  publicationsByThemeAndTopic,
```

Comment:
This seems quite wordy and puts the onus on the component consumer to group the publications first (seems like a bit of a hassle). Couldn't we just do the grouping within this component?

Reply:
Yup, that makes it much cleaner and better encapsulated, agreed.  Done!